configure.ac

```diff
@@ -123,6 +123,8 @@ AC_C_BIGENDIAN
 
 GCC_AS_CFI_PSEUDO_OP
 
+AC_ARG_VAR([WASM64_MEMORY64], [Used only for the wasm64 target. Set to 1 (default) or 2 for Emscripten's -sMEMORY64 mode])
```

Comment:
What's the difference between these?

Reply:
The user can use `WASM64_MEMORY64` to control Emscripten's [`-sMEMORY64` flag](https://emscripten.org/docs/tools_reference/settings_reference.html#memory64). Both values 1 and 2 enable 64bit pointers for clang/lld. Emscripten's `-sMEMORY64=2` additionally lowers the output to wasm32 in Binaryen so it runs on [engines that don't support wasm64](https://webassembly.org/features/).